src/data/set/pointwise.lean

```diff
@@ -54,6 +54,16 @@ set multiplication, set addition, pointwise addition, pointwise multiplication,
 pointwise subtraction
 -/
 
+/--
+Pointwise monoids (`set`, `finset`, `filter`) have derived pointwise actions of the form
+`has_scalar α β → has_scalar α (set β)`. When `α` is `ℕ` or `ℤ`, this action conflicts with the
+nat or int action coming from `set β` being a `monoid` or `div_inv_monoid`.
```

Comment:
Can you add the `2 • {a, b}` as `{2 • a, 2 • b}` vs `{a + a, a + b, b + b}` example

Reply:
Adding `{a + a, a + b, b + a, b + b}` instead, to avoid assuming commutativity and because it wouldn't be defeq anyway.